### DISCUSSION TITLE
Welcome to HUST Mirrors Discussions!

### DISCUSSION THREAD

**Discussion by mudongliang** [hust-open-atom-club/hust-mirrors#108] [Announcements]
Start dicussion here

**Comment by M0rtzz** [comment]
@mudongliang 

May I ask how were these icons generated:

![image](https://github.com/user-attachments/assets/3db9dfc4-348d-4c81-927d-8520f13defaa)

![image](https://github.com/user-attachments/assets/f0b32a5e-eab7-421a-a0ec-72e77877fbf5)

Is it detected that the mirror has corresponding documents or can be automatically generated using `hustmirror-cli`?

PS: What is the purpose of [this `cname` in docs](https://github.com/hust-open-atom-club/hust-mirrors/blob/master/docs/linuxmint.md?plain=1#L4), Is it used to customize domain?

**Comment by jxlpzqc** [comment]
First, these icons are generated using list of all docs files and `meta.config.ts`. Whether the document exsist is saved in previous and whether it can be generated by `hustmirror-cli` is saved in latter. 

Btw, the `cname` tag seems meaningless in current version, it may be kept from historical version.

**Reply by M0rtzz** [reply]
> First, these icons are generated using list of all docs files and `meta.config.ts`. Whether the document exsist is saved in previous and whether it can be generated by `hustmirror-cli` is saved in latter.
> 
> Btw, the `cname` tag seems meaningless in current version, it may be kept from historical version.

Thank you🥰.

What are the meanings of these variables below?

[var](https://github.com/hust-open-atom-club/hust-mirrors/blob/master/meta.config.ts#L6)

[var](https://github.com/hust-open-atom-club/hust-mirrors/blob/master/meta.config.ts#L8)

[var](https://github.com/hust-open-atom-club/hust-mirrors/blob/master/meta.config.ts#L10-L11)

[var](https://github.com/hust-open-atom-club/hust-mirrors/blob/master/meta.config.ts#L13)
